.github/workflows/remote_package_install.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10","3.12"]
+        python-version: ["3.10","3.12","3.13"]
```

Comment:
intent with skipping 3.11 was to test lower & highest supported version. should we continue that pattern?